libshaderc_util/src/compiler.cc

```diff
@@ -102,6 +103,29 @@ EShMessages GetMessageRules(shaderc_util::Compiler::TargetEnv env,
 
 namespace shaderc_util {
 
+unsigned int GlslangInitializer::initialize_count_ = 0;
+std::mutex GlslangInitializer::mutex_;
```

Comment:
This doesn't add a global constructor does it?

Reply:
std::mutex is a class, so I think it will and that will aggravate the chromium devs. I will write a quick patch to handle this differently.

Reply:
New version is up, avoids the global construction, but does cause a small memory leak. Classic Iron Triangle situation here, we get to pick 2 of thread safe, no global constructors, and no memory leaks. The memory leak is of a single std::mutex, so I don't think it is a major issue.